config.go

```diff
@@ -60,6 +70,19 @@ func (cfg Config) MakeLabels(types []string) string {
 	return labels
 }
 
+func (cfg Config) SavePropertyNativeType(node graph.Node, tp, val string) interface{} {
```

Comment:
Call tp  -> expandedpropertyKey
Call this func with expanded property key, so don't keep calling cfg.Expand